src/main/java/com/ekart/thunderbolt/controller/ProductController.java

```diff
@@ -0,0 +1,126 @@
+package com.ekart.thunderbolt.controller;
+
+
+import com.ekart.thunderbolt.DAO.ProductDAO;
+import com.ekart.thunderbolt.service.ProductService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.AllArgsConstructor;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+import static org.springframework.http.ResponseEntity.status;
+
+@RestController
+@RequestMapping("/api/product")
+@AllArgsConstructor
+@Tag(name="Product" , description = "Product APIs")
+public class ProductController {
+
+    private final Logger LOGGER = LoggerFactory.getLogger(ProductController.class);
+    private final ProductService productsService;
+
+    /**
+     * This API is used to create a new product in Thunderbolt.
+     * */
+    @PostMapping
+    @Operation(
+            summary = "Create a new Product",
+            responses = {
+                    @ApiResponse( description = "Create product", responseCode = "200", content = @Content(mediaType = "application/json", schema = @Schema(implementation = ProductDAO.class))),
+                    @ApiResponse( description = "Unable to create product", responseCode = "400", content = @Content(mediaType = "application/json"))
```

Comment:
Also add the other default possible statuses, for now 500 Internal Server Error. And once we bring this authentication and authorization, we will add 401 Unauthorized and 403 Forbidden.

Reply:
Description should be Bad Request. Unable to create product seems more like an error from your side, whereas 400 is generally used for input error.